src/i18n/en.json

```diff
@@ -316,22 +538,256 @@
+        "LocationDidntFindItem": "location didn't find item",
+        "ExpectedCultItemError": "Expected Cult item",
+        "CantFindRunesCompendiumError": "Can't find runes Compendium {runesCompendiumName}",
+        "CantFindCultRuneError": "Can't find cult rune {newRune} among allRunesIndex",
+        "CantFindRuneInAllRunesCompendiumError": "Can't find rune in all runes compendium",
+        "ItemWasNotHitLocationError": "Called hitLocation#onActorPrepareEmbeddedEntities on an item that wasn't a hitLocation",
+        "HitLocationDoesNotHaveActorError": "Hit Location item didn't have an actor",
+        "ActorDoesNotHaveMaxHpError": "Actor doesn't have max hitPoints",
+        "OpposingRuneDoesNotExistWarning": "Opposing Rune doesn't exist",
+        "WrongItemTypeRuneMagicError": "RQG | Wrong itemtype or not embedded item in Actor PrepareEmbeddedEntities",
+        "ActorDoesNotHaveCultOnRuneMagicWarning": "RQG | Cult referenced by rune magic item ${item.name} doesn't exist on actor ${actor.name}",
+        "PrepareDerivedDataNotSkillError": "Calling Skill#onActorPrepareDerivedData with something else than a skill",
+        "ActorNotCharacterError": "Actor is not of type character",
+        "CantFindWeaponSkillWarning": "Can't find weapon skill",
+        "NoWeaponSkillFromSkillOriginWarning": "No weapon skill with from skillOrigin \"{skillOrigin}\".",
+        "CantFindSkillAssociatedWithWeaponWarning": "Can't find the Skill associated with this weapon.",
+        "CantCreateActiveEffect": "Can't create Active Effect",
+        "NoIdOnTokenError": "No ID on Token",
+        "NoIdOnActorError": "No ID on Actor",
+        "ItemWithIdDoesNotHaveChanceError": "Can't find an item with ID {itemId} on actor named {actorName} to show item chat card.",
+        "CouldNotFindActorByIdsWarn": "Could not find actor by actor ID {actorId} or tokenID {tokenId} to do ability roll."
+      },
+      "Passion": {
+        "Name": "Name",
+        "Passion": "Passion",
+        "Subject": "Subject",
+        "Chance": "Chance",
+        "Experience": "Experience",
+        "BackStory": "Back Story",
+        "PassionEnum": {
+          "Ambition": "Ambition",
+          "Cowardly": "Cowardly",
+          "Devotion": "Devotion",
+          "Fear": "Fear",
+          "Hate": "Hate",
+          "Honor": "Honor",
+          "Loyalty": "Loyalty",
+          "Love": "Love",
+          "Gluttony": "Gluttony",
+          "Vanity": "Vanity"
+        }
+      },
+      "Rune": {
+        "Name": "Name",
+        "Rune": "Rune",
+        "Type": "Type",
+        "HasExperienceQ": "Has Experience?",
+        "OpposingRune": "Opposing Rune",
+        "MinorRunes": "Minor Runes",
+        "MasteredSorceryQ": "Mastered (Sorcery)?",
+        "RuneType": {
+          "element": "Element",
+          "power": "Power",
+          "form": "Form",
+          "condition": "Condition",
+          "technique": "Technique"
+        }
+      },
+      "RuneMagic": {
```

Comment:
Break out most of this to "Spell" since there is now an [interface](https://github.com/sun-dragon-cult/fvtt-system-rqg/blob/master/src/data-model/item-data/spell.ts) that is common to both Spirit, Rune (and eventually Sorcery) Magic.

Reply:
Fixed